### PR TITLE
fix: add configuration disk capacity config for lru and fix some bug

### PR DESCRIPTION
**Pull request by chyezh**
issue: #30361

- Add configurable disk capacity limit

- fix bitset reset logic

- make insert record reinsert after clear

### CODE REVIEW

File: internal/core/src/segcore/SegmentSealedImpl.cpp [hunk 1]
```diff
@@ -1125,9 +1125,9 @@ SegmentSealedImpl::bulk_subscript_impl(int64_t element_sizeof,
 
 void
 SegmentSealedImpl::ClearData() {
-    field_data_ready_bitset_.clear();
-    index_ready_bitset_.clear();
-    binlog_index_bitset_.clear();
+    field_data_ready_bitset_.reset();
+    index_ready_bitset_.reset();
+    binlog_index_bitset_.reset();
```
Comment (chyezh) [inline internal/core/src/segcore/SegmentSealedImpl.cpp:1130]: reset will reset the bitset into zero value for every bit.
clear will clear the underlying allocated space.
we need `reset` here.

**Review by congqixia — COMMENTED** [review]
/approve
but the setting `is_sealed` is too confusing here

**Comment by chyezh** [comment]
> /approve but the setting `is_sealed` is too confusing here

`is_sealed` is the field of `insert_record_` but not sealed segment.
insert operation of `insert_record_`  will be called when load data of sealed segment.

**Comment by chyezh** [comment]
It's a part of fix for issue: #32135

**Pull request merged**